tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.h

```diff
@@ -0,0 +1,102 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// This file contains nccl api for alpa to use. 
+
+#ifndef TENSORFLOW_COMPILER_XLA_SERVICE_GPU_NCCL_UTILS_H_
+#define TENSORFLOW_COMPILER_XLA_SERVICE_GPU_NCCL_UTILS_H_
+
+#include <memory>
+
+#include "tensorflow/compiler/xla/status.h"
+#include "tensorflow/compiler/xla/statusor.h"
+#include "tensorflow/compiler/xla/xla_data.pb.h"
+
+// Common place for all collective thunks to include nccl/rccl headers.
+#if TENSORFLOW_USE_ROCM
+#include "rocm/include/rccl/rccl.h"
+#else
+#include "third_party/nccl/nccl.h"
+#endif
+
+#include "pybind11/pybind11.h"
+#include "pybind11/attr.h"
+#include "pybind11/cast.h"
+#include "pybind11/numpy.h"
+#include "pybind11/pytypes.h"
+#include "pybind11/stl_bind.h"
+
+#include "tensorflow/compiler/xla/python/py_buffer.h"
+#include "tensorflow/compiler/xla/python/py_client.h"
+
+namespace xla {
+namespace alpa_nccl {
```

Comment:
Following the code style, this should be 
```c++
namespace xla {
namespace gpu {
```
Did you meet any conflicts when using `xla::gpu` namespace?